_data/versions.yml

```diff
@@ -32,15 +32,19 @@
       release_notes: "/article/dev-snapshot-godot-4-2-dev-1/"
 
 - name: "4.1.3"
-  flavor: "rc1"
-  release_date: "27 October 2023"
-  release_notes: "/article/release-candidate-godot-4-1-3-rc-1/"
+  flavor: "stable"
+  release_date: "1 November 2023"
```

Comment:
```suggestion
  release_date: "2 November 2023"
```

Reply:
I used the release date when binaries were published, not the date of the article :P